src/views/RichWorkspace.vue

```diff
@@ -107,6 +107,7 @@ export default {
 		}
 		subscribe('Text::showRichWorkspace', this.showRichWorkspace)
 		subscribe('Text::hideRichWorkspace', this.hideRichWorkspace)
+		subscribe('files:node:deleted', this.onFileDeleted)
```

Comment:
Please also unsubscribe in beforeDestroy :) Otherwise this looks good 👍

Reply:
Ah and maybe we should also cover the `files:node:renamed` event to cover file names being changed from/to rich workspace filenames (reference for the possible filenames https://github.com/nextcloud/text/blob/23ce8f876852771accba68bad2c297271fb3aa22/lib/Service/WorkspaceService.php#L14-L43).